frontend/.env.example

```
@@ -0,0 +1,11 @@
+# Stellar Network Configuration
+VITE_STELLAR_NETWORK=testnet
+VITE_SOROBAN_RPC_URL=https://soroban-testnet.stellar.org
+VITE_NETWORK_PASSPHRASE=Test SDF Network ; September 2015
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_NETWORK_PASSPHRASE key should go before the VITE_SOROBAN_RPC_URL key

(UnorderedKey)